apps/design-land/src/app/radio/radio.module.ts

```diff
@@ -1,60 +1,20 @@
-import {
-  NgModule,
-  ComponentFactoryResolver,
-  Injector,
-} from '@angular/core';
-import {
-  DaffRadioModule,
-  DaffButtonModule,
-} from '@daffodil/design';
-import { ReactiveFormsModule } from '@angular/forms';
 import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
 
-import { RadioComponent } from './radio.component';
-import { DesignLandRadioRoutingModule } from './radio-routing.module';
-import { RADIO_EXAMPLES } from '@daffodil/design/radio/examples';
-import {
-  createCustomElement,
-  NgElementConstructor,
-} from '@angular/elements';
 import { DesignLandExampleViewerModule } from '../core/code-preview/container/example-viewer.module';
-
-
-
-export interface CustomClassElement<T> {
-  element: NgElementConstructor<T>;
-  class: T;
-}
+import { DesignLandRadioRoutingModule } from './radio-routing.module';
+import { RadioComponent } from './radio.component';
 
 @NgModule({
   declarations: [
     RadioComponent,
-    ...RADIO_EXAMPLES,
   ],
   imports: [
     DesignLandExampleViewerModule,
     DesignLandRadioRoutingModule,
-    DaffRadioModule,
-    ReactiveFormsModule,
     CommonModule,
   ],
 })
 export class RadioModule {
```

Comment:
Can we update this to `DesignLandRadioModule` and do it in the component too since you started doing that to other components in the PR?

Reply:
fixed.

Reply:
fixed.